auslib/web/admin/views/permissions.py

```diff
@@ -44,7 +44,8 @@ def get(self, username):
                                detail="You are not authorized to view permissions of other users.")
 
         permissions = dbo.permissions.getUserPermissions(username)
-        if not permissions:
+
+        if username != current_user and not permissions:
```

Comment:
This fixes the bug, but it actually introduces a new one: if a User without permissions queries for their own permissions by name (eg: a GET to "/users/vikas"), a 200 is returned instead of a 404. We _only_ want a 200 when no permissions exist for the specific case of "/users/current". If a user is explicitly named, we should return a 404 if they have no permissions.